js/server/modules/@arangodb/aql-helper.js

```diff
@@ -381,20 +381,15 @@ function removeClusterNodesFromPlan (nodes) {
   });
 }
 
-function roundCost (obj) {
+function removeCost (obj) {
```

Comment:
Maybe put a comment here that says that this function recursively removes any keys `estimatedCost` or `selectivityEstimate`, and that this is done to avoid triggering errors in tests that `AssertEqual` on floating point values (and that in the cases we test we don't care about these values)